ofl/pinyonscript/DESCRIPTION.en_us.html

```diff
@@ -1,7 +1,12 @@
-<p><strong>Foundry: <a href="http://www.sorkintype.com">Sorkin Type Co</a></strong></p>
+<p>
+<strong>Foundry: <a href="http://www.sorkintype.com">Sorkin Type Co</a></strong></p>
 
-<p>Pinyon Script is a romantic round hand script style font. It also sports swashes which are confident and showy somehow giving the type a feeling suggestive of the American west. Perhaps this is why, despite refinement and aristocratic style; Pinyon Script manages to feel so friendly. Nicole Script uses a high stroke contrast and is very slanted making it most suitable for use at larger sizes.</p>
+<p>Pinyon Script is a romantic round hand script style font. It also sports swashes which are confident and showy 
+somehow giving the type a feeling suggestive of the American west. 
+Perhaps this is why, despite refinement and aristocratic style; Pinyon Script manages to feel so friendly. 
+Pinyon Script uses a high stroke contrast and is very slanted making it most suitable for use at larger sizes.</p>
 
 <p>Source files are available from <a
 href="http://code.google.com/p/googlefontdirectory/">Google Code</a>.
```

Comment:
This link should be removed. @ebensorkin is there a github repo for that project?